solutions/dedicated/README.md

```diff
@@ -0,0 +1,13 @@
+# IBM Cloudant Dedicated on IBM Cloud Dedicated
+
+This architecture creates an IBM Cloudant Dedicated instance in IBM Cloud Dedicated and supports provisioning the following resources:
+
+- A resource group, if one is not passed in.
+- An IBM Cloudant Dedicated instance on IBM Cloud Dedicated
+- A IBM Cloudant database
+
+![dedicated-cloudant](../../images/dedicated_cloudant.svg)
```

Comment:
Can you add the diagram into a folder called `reference-architectures` instead. This is where they will need to be for when we publish a public DA.